dev-lang/solidity/solidity-0.4.2.ebuild

```diff
@@ -1,38 +0,0 @@
-# Copyright 1999-2017 Gentoo Foundation
-# Distributed under the terms of the GNU General Public License v2
-
-EAPI=6
```

Comment:
@MathyV please separate ebuild bump and removal into two separate commits (facilitates reverting).